COLLADASaxFrameworkLoader/src/COLLADASaxFWLMeshLoader.cpp

```diff
@@ -892,8 +900,7 @@ namespace COLLADASaxFWL
                 SourceBase* sourceBase = getSourceById ( sourceId );
                 if ( sourceBase == 0 ) 
                 {
-                    handleFWLError ( SaxFWLError::ERROR_DATA_NOT_VALID, "SourceBase of tex coords with semantic TEXCOORD not valid!" );
-                    return;
+                    return handleFWLError ( SaxFWLError::ERROR_DATA_NOT_VALID, "SourceBase of tex coords with semantic TEXCOORD not valid!" );
```

Comment:
Could be, but wouldn't fixing the message be actually another patch? Unrelated to this pull request?